prime_server/http_protocol.hpp

```diff
@@ -190,4 +194,6 @@ struct http_response_t : public http_entity_t {
 
 using http_server_t = server_t<http_request_t, http_request_info_t>;
 
+shortcircuiter_t<http_request_t> make_shortcircuiter(const std::string& health_check_path  = "/health_check",
+                                                     const uint8_t& verb_mask = std::numeric_limits<uint8_t>::max());
```

Comment:
while using `max` will work, i think its a bit hard to understand, i would advocate for being more explicit (its easier to read then):

```suggestion
constexpr uint8_t ALL_VERBS_MASK = OPTIONS | GET | HEAD | POST | PUT | DELETE | TRACE | CONNECT;

shortcircuiter_t<http_request_t> make_shortcircuiter(const std::string& health_check_path  = "/health_check",
                                                     const uint8_t& verb_mask = ALL_VERBS_MASK);
```